examples/index.json

```diff
@@ -905,6 +905,13 @@
         "verifyOutput": true,
         "isLearnByExample": true
       },
+      {
+        "name": "On conflict policy",
```

Comment:
lets remove the `policy` word here.

Reply:
we need to remove all occurrences of `policy`

Reply:
Since we already have`BBE` named `On conflict clause` we can't just remove the `policy` word. We need to think of a better name If `On conflict policy` is not the best match